### ISSUE TITLE
Bug integrating Cesium into a vite-plugin-ssr application

### ISSUE THREAD

**Issue by davenquinn** [vikejs/vike#1139]
### Description

I have been working to integrate a [CesiumJS](https://cesium.org) viewer into an existing vite-plugin-ssr application, via [Macrostrat's Cesium viewer module](https://github.com/UW-Macrostrat/cesium-viewer). I am early in the debugging process of this (it's a major change in bundling strategy from Webpack, as well. But I encountered the below error and am reporting it here as asked! Any insights are appreciated.

```
Error: [vite-plugin-ssr@0.4.142][Bug] You stumbled upon a bug in vite-plugin-ssr's source code. Go to https://github.com/brillout/vite-plugin-ssr/issues/new and copy-paste this error; a maintainer will fix the bug (usually under 24 hours).
    at getFilePathVite (file:///Users/Daven/Projects/Macrostrat/Software/web/.yarn/__virtual__/vite-plugin-ssr-virtual-ea735fca57/0/cache/vite-plugin-ssr-npm-0.4.142-bb82084716-b89f194747.zip/node_modules/vite-plugin-ssr/dist/esm/utils/getFilePathAbsolute.js:57:5)
    at getPrettyErrorWithCodeSnippet (file:///Users/Daven/Projects/Macrostrat/Software/web/.yarn/__virtual__/vite-plugin-ssr-virtual-ea735fca57/0/cache/vite-plugin-ssr-npm-0.4.142-bb82084716-b89f194747.zip/node_modules/vite-plugin-ssr/dist/esm/node/plugin/shared/loggerNotProd/errorWithCodeSnippet.js:49:24)
    at logErr (file:///Users/Daven/Projects/Macrostrat/Software/web/.yarn/__virtual__/vite-plugin-ssr-virtual-ea735fca57/0/cache/vite-plugin-ssr-npm-0.4.142-bb82084716-b89f194747.zip/node_modules/vite-plugin-ssr/dist/esm/node/plugin/shared/loggerNotProd.js:83:29)
    at logViteErrorContainingCodeSnippet (file:///Users/Daven/Projects/Macrostrat/Software/web/.yarn/__virtual__/vite-plugin-ssr-virtual-ea735fca57/0/cache/vite-plugin-ssr-npm-0.4.142-bb82084716-b89f194747.zip/node_modules/vite-plugin-ssr/dist/esm/node/plugin/shared/loggerNotProd.js:61:5)
    at Object.config.logger.<computed> [as error] (file:///Users/Daven/Projects/Macrostrat/Software/web/.yarn/__virtual__/vite-plugin-ssr-virtual-ea735fca57/0/cache/vite-plugin-ssr-npm-0.4.142-bb82084716-b89f194747.zip/node_modules/vite-plugin-ssr/dist/esm/node/plugin/shared/loggerVite.js:37:13)
    at file:///Users/Daven/Projects/Macrostrat/Software/web/.yarn/__virtual__/vite-virtual-23cbd68f33/0/cache/vite-npm-4.4.9-e845c1bbf8-c511024cea.zip/node_modules/vite/dist/node/chunks/dep-df561101.js:42035:43
```

**Comment by brillout** [comment]
Does `DEBUG=vike:error` as described at https://vike.dev/errors#verbose solve the situation?

**Comment by brillout** [comment]
Can you provide a reproduction? That'd be the best for this particular bug.

**Comment by davenquinn** [comment]
I will check!

**Comment by davenquinn** [comment]
I did the due diligence on my side to ensure that the app worked in Vite _sans_ Vike, and discovered that in fact there were quite a few underlying compilation errors due to ESM module difficulties. Once I solved those outside of the SSR context, this error was resolved.

This clearly is not an internal issue with Vike, but I wonder if the Vike-specific compilation steps are swallowing errors they should instead forward, perhaps in the service of this "user-friendly" warning. It'd help if the default was a little more verbose, probably.

**Comment by brillout** [comment]
I had a closer look at this: it actually is an internal issue with Vike.

I've pre-released a new version `vike@0.4.142-commit-8bf568b` which includes debug logs for this issue.

Can you give me the error again `Error: [vite-plugin-ssr@0.4.142][Bug] You stumbled upon a bug in ...` using that version? This time, the error message will include debug information which will allow me to get to the bottom of this.
